src/chains/models.py

```diff
@@ -129,6 +130,9 @@ def get_disabled_wallets(self) -> QuerySet["Wallet"]:
     def __str__(self) -> str:
         return f"{self.name} | chain_id={self.id}"
 
+    class Meta:
+        permissions = [("change_only_warning", "Can change only warning")]
```

Comment:
`only` is included in the name because of the nature of this permission. This permission will imply two things for an user: 
- The user can edit the `warning` field.
- The user cannot edit any other `Chain` field, i.e. the rest of the field will be marked as read-only.

I know it is not very intuitive, but we agreed on taking this approach to avoid using a third party dependency which would let us define field-level permissions. Even using that dependency, I suspect the outcome wouldn't be very intuitive either, because our goal is not to grant new permissions, but preventing some users to access some fields.